sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIAuthenticationAudience.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using static System.Net.WebRequestMethods;
+
+namespace Azure.AI.OpenAI;
+
+/// <summary>
+/// Represents cloud authentication audiences available for Azure OpenAI.
+/// These audiences correspond to authorization token authentication scopes.
+/// </summary>
+public readonly partial struct AzureOpenAIAuthorizationAudience : IEquatable<AzureOpenAIAuthorizationAudience>
+{
+    private readonly string _value;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AzureOpenAIAuthorizationAudience"/> object.
+    /// </summary>
+    /// <remarks>
+    /// Please consider using one of the known, valid values like <see cref="AzurePublicCloud"/> or <see cref="AzureGovernmentCloud"/>.
+    /// </remarks>
+    /// <param name="value">
+    /// The Azure Active Directory audience to use when forming authorization scopes.
+    /// For Azure OpenAI, this value corresponds to a URL that identifies the Azure cloud where the resource is located.
+    /// For more information: <see href="https://docs.microsoft.com/azure/azure-government/documentation-government-cognitiveservices" />.
+    /// </param>
+    /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+    public AzureOpenAIAuthorizationAudience(string value)
+    {
+        Argument.AssertNotNullOrEmpty(value, nameof(value));
+        _value = value;
+    }
+
+    private const string AzurePublicCloudValue = "https://cognitiveservices.azure.com";
+    private const string AzureGovernmentCloudValue = "https://cognitiveservices.azure.us";
+
+    /// <summary>
+    /// The authorization audience used to connect to the public Azure cloud. Default if not otherwise specified.
+    /// </summary>
+    public static AzureOpenAIAuthorizationAudience AzurePublicCloud { get; } = new AzureOpenAIAuthorizationAudience(AzurePublicCloudValue);
+
+    /// <summary>
+    /// The authorization audience used to authenticate with the Azure Government cloud.
+    /// </summary>
+    /// <remarks>
+    /// For more information, please refer to
+    /// <see href="https://learn.microsoft.com/azure/azure-government/documentation-government-cognitiveservices" />.
+    /// </remarks>
+    public static AzureOpenAIAuthorizationAudience AzureGovernmentCloud { get; } = new AzureOpenAIAuthorizationAudience(AzureGovernmentCloudValue);
```

Comment:
Looking at other instances, this is simply called "AzureGovernment" instead of "AzureGovernmentCloud"